Desktop version first implementation

s2jack opened the pull request:
Add background images for about, home and contact section
Resize backgrounds and texts
Add new css file for desktop version
Link desktop css in html head
Add new divs for new backgrounds
Add new inputs for firstname and lastname
Add new grid tempplate for recent works cards container
Declare new grid-row and column numbers
Fix margins and paddings
Fix font sizes and colors

s2jack committed:
Add desktop css file

Add desktop css link in html

s2jack committed:
Declare Desktop version

Add background images for about, home and contact section
Resize backgrounds and texts
Add new css file for desktop version
Link desktop css in html head
Add new divs for new backgrounds
Add new inputs for firstname and lastname
Add new grid tempplate for recent works cards container
Declare new grid-row and column numbers
Fix margins and paddings
Fix font sizes and colors

s2jack committed:
Fix linter errors

skyv26 commented on desktop_styles.css:
```diff
@@ -0,0 +1,309 @@
+  width: 62vw;
+}
+
+#works {
+  height: auto;
+  top: 73.5vw;
+  width: 100vw;
+}
```
- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching with original. Please have a look at the below comparison table to understand the problem in a better way through visualization, according to which your **work-section** should be responsive and looks perfect on screen **>=768px**. Please adhere to the project requirement and implement the things properly.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-08 00-00-29](https://github.com/s2jack/create-portfolio-website1/assets/66476329/c022d82a-58ed-4fe0-b1d1-ec8f73ec273a)

</td>

<td>

![Screenshot from 2023-06-07 23-59-18](https://github.com/s2jack/create-portfolio-website1/assets/66476329/5f960921-6a54-4228-8f83-47920f1d6161)


</td>
</tr>
</table>


#### Requirement

![Screenshot from 2023-06-08 00-03-31](https://github.com/s2jack/create-portfolio-website1/assets/66476329/2b583e12-d26a-4d3a-90b4-92fd8cd4df18)

s2jack replied:
there is only a design for the 1440px width property. What should I do? especially for the works section. I really don't know how we can do it.  I guess, you need to compare my design and the Figma template for 1440px screen width.

skyv26 replied:
Hi @s2jack, Figma design is just for reference and I already have attached the project requirement according to which you must have to implement the *Figma* design no matter *1440px* on >= 768px screen size. Because I am pretty confident that you used all the CSS styles for desktop design from Figma which is why your design is fixed for 1440px screen size but you shouldn't follow the Figma styles except the colour, font family, hover effect and a few others according to what makes sense. Now, the answer to your problem. Well, if you don't know then it is the right time to take a challenge for yourself and try to learn and implement the changes as I requested because in the upcoming projects especially during **Final Capstone Project** you will even not get the `Figma` design you will only get a design which nothing but a photo on behance website, which you need to follow in order to complete project. So if it is difficult for you now then it will be difficult during Final Capstone project too, but if you learn and implement the things now then *Final Capstone Project* will be like a cat walk. Any other question please feel free to mention @skyv26 and ask the question without hesitation, I am open to answer all your doubts with best possible suggestions, advices and with 100% satisfactory solution.

skyv26 commented on desktop_styles.css:
```diff
@@ -0,0 +1,309 @@
+  height: auto;
+  top: 2331px;
+  width: 100vw;
+}
```
- [x] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching with original. Please have a look at the below comparison table to understand the problem in a better way through visualization, according to which your **about-section** should be responsive and looks perfect on screen **>=768px**. Please adhere to the project requirement and implement things properly.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-08 00-00-44](https://github.com/s2jack/create-portfolio-website1/assets/66476329/f8f192b1-a8b0-4e8e-b3e1-83a074758425)

</td>

<td>

![Screenshot from 2023-06-07 23-59-30](https://github.com/s2jack/create-portfolio-website1/assets/66476329/17cf8e1c-b996-4454-b95c-4765b52c8fbd)


</td>
</tr>
</table>

- [x] Please also reduce the gap between the About section and the Work section because your design right now looks unprofessional.

### Gap Issue

![Screenshot from 2023-06-08 00-09-40](https://github.com/s2jack/create-portfolio-website1/assets/66476329/cde8e92b-cc89-4526-afa7-e96d08b60d6c)


#### Requirement

![Screenshot from 2023-06-08 00-03-31](https://github.com/s2jack/create-portfolio-website1/assets/66476329/2b583e12-d26a-4d3a-90b4-92fd8cd4df18)

s2jack replied:
The screenshots which you compared above belong to 1440px width.  I will fix the gap issue. But I need to figure out the other requests. I need to ask the student success team and my coding partners.

skyv26 replied:
@s2jack Sure I would suggest you please if you think that this code review is not fair, you can find a link for a second opinion which is inside my `PR` summary at the bottom.

skyv26 commented on desktop_styles.css:
```diff
@@ -0,0 +1,309 @@
+  justify-content: center;
+}
+
+#contact {
```
- [x] Kindly stick with the *Figma* design as closely as possible, it seems like your design is not matching with original. Please have a look at the below comparison table to understand the problem in a better way through visualization, according to which your **contact-section** should be responsive and looks perfect on screen **>=768px**. Please adhere to the project requirement and implement things properly.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-08 00-00-49](https://github.com/s2jack/create-portfolio-website1/assets/66476329/e8eeb217-f507-4b2a-a886-2fe01618aa2e)

</td>

<td>

![Screenshot from 2023-06-07 23-59-39](https://github.com/s2jack/create-portfolio-website1/assets/66476329/c790163e-f36a-43b8-948e-f52fd0b49966)


</td>
</tr>
</table>


#### Requirement

![Screenshot from 2023-06-08 00-03-31](https://github.com/s2jack/create-portfolio-website1/assets/66476329/2b583e12-d26a-4d3a-90b4-92fd8cd4df18)

skyv26 commented on styles.css:
- [x] Please stick with the design and make it responsive below < 768px screen size as I can see that your design is not adapting or taking 100% of the screen size below the **768px** screen which makes your design looks unprofessional. Kindly use **width: 100%** at the place of either **width: 327px** or **width: 375px** or remove the fixed width because by default the width for the block element is always **100%**.

### Demo Video

[Screencast from 08-06-23 12:25:55 AM IST.webm](https://github.com/s2jack/create-portfolio-website1/assets/66476329/8e3cb276-7016-4a18-9e0a-afe0fc508a5f)

skyv26 reviewed (changes requested):
## Hi @s2jack, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZDMzYmY2ZWU0MmEwMTJjZTczMGVhNDhlM2E2NWViYmQ0YzY5ZTBiNiZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/17FxSFyYNOgThnonDK/giphy.gif)

## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 

## Required Changes ♻️

- [ ] Please 🙏🏻  add the `MIT` license properly because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .

### Your Repo License

![image](https://github.com/s2jack/create-portfolio-website1/assets/66476329/09c522aa-607e-4203-9ed6-950b65d928f2)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   <img src="https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white" alt="@skyv26" />
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   <img src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" alt="linkedin-skyv2022" />
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   <img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="@vrma_aakash" />
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   <img src="https://img.shields.io/badge/Instagram-E4405F?style=for-the-badge&logo=instagram&logoColor=white" alt="@skybrel" />
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   <img src="https://user-images.githubusercontent.com/66476329/243158690-61eb1021-fbb6-464d-b841-a09c4caa156e.png" alt="@skybrel" />
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

s2jack committed:
Fix padding and margins

s2jack committed:
Add animation and transition

Add scale-anim class to the menu items and the social media icons
Declare scale-animation properties in desktop-styles.css

s2jack committed:
Fix all margins, sizes and paddings

Convert all pixels properties with viewport width

s2jack committed:
Fix conflict errors

s2jack committed:
Fix stylelint errors

s2jack committed:
Add MIT licence to my ReadME file

algerina commented on README.md:
```diff
@@ -150,6 +150,26 @@ I would like to thank Microverse Family and all my coding partners.
 ## 📝 License <a name="license"></a>
 
-This project is not licensed.
+MIT License
+
+Copyright (c) [year] [fullname]
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
- [ Optional] You did well implementing the required changes however, I recommend adding the MIT license link to your README and not the license itself.

algerina reviewed (approved):
Hi @s2jack  ,

Great job so far! 

May I highlight the following: 
- [x] Your portfolio looks awesome👏  ✔️ 
- [x] You implemented all the required changes by the previous TSE's✔️ 



## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:

<img src="https://media4.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>

 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_
